docs/src/tutorials/yocto-linux.md

```diff
@@ -51,61 +50,191 @@ process. If using these options, to see previous commits or other branches see
+
+In `poky/build/conf/bblayers.conf`:
 
 ```
+# POKY_BBLAYERS_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+POKY_BBLAYERS_CONF_VERSION = "2"
+
+BBPATH = "${TOPDIR}"
+BBFILES ?= ""
+
+BBLAYERS ?= " \
+  /home/yocto/poky/meta \
+  /home/yocto/poky/meta-poky \
+  /home/yocto/poky/meta-yocto-bsp \
+  /home/yocto/meta-openembedded/meta-oe \
+  /home/yocto/meta-openembedded/meta-python \
+  /home/yocto/meta-openembedded/meta-networking \
+  /home/yocto/meta-tedge \
+  "
+```
+
+### Use systemd init manager
 
-### Configure the build
+For thin-edge to work, it has to run some setup scripts upon the first boot and interact with system services via the
+init system. Right now, `meta-tedge` only supports systemd init manager. The default in yocto is initrd, thus we have to
+change it.
 
-Activate `Systemd` as default init manager by adding following line to `local.conf`:
+> This, and any successive changes to the build configuration should happen only in your local configuration, ie.
+> `poky/build/conf/local.conf` or in configuration files of layers you create yourself. Changing any files in layers you
+> do not control - in our example `meta-tedge` or any of the layers in `meta-openembedded` - is discouraged, because any
+> changes you make to them may be lost when you update the layer.
+
+Activate `systemd` as default init manager by adding following line to `poky/build/conf/local.conf`:
 
 ```
 INIT_MANAGER="systemd"
 ```
 
-Build `tedge` by running following command :
+### Use apt package manager (optional)
 
-```bash
-$ bitbake tedge-full-image
+The yocto distribution includes a package manager for installing or removing packages during runtime. Let us use apt
+package manager:
+
+In `poky/build/conf/local.conf` find the following section and change `PACKAGE_CLASSES` from `package_rpm` to `package_deb`
+as such:
+
+```
+#
+# Package Management configuration
+#
+# This variable lists which packaging formats to enable. Multiple package backends
+# can be enabled at once and the first item listed in the variable will be used
+# to generate the root filesystems.
+# Options are:
+#  - 'package_deb' for debian style deb files
+#  - 'package_ipk' for ipk files are used by opkg (a debian style embedded package manager)
+#  - 'package_rpm' for rpm style packages
+# E.g.: PACKAGE_CLASSES ?= "package_rpm package_deb package_ipk"
+# We default to rpm:
+PACKAGE_CLASSES ?= "package_deb"
 ```
 
-### Run the build
+### Build and run
 
-When the build it complete, run it with qemu:
+Finally, build the image and run it in the emulator.
+
+Build `tedge` by running following command:
 
 ```bash
-$ runqemu nographic
+bitbake tedge-full-image
 ```
 
+When the build it complete, run it with qemu:
 
-## Further recommendations
-
-After building the reference distribution and image, you can explore creating your own layer and image, and then
-integrating `tedge-*` recipes for it. 
+```bash
+runqemu nographic
+```
```

Comment:
Mention from where the this command to be run

Reply:
It can work from any directory, which is now included in the guide.